src/controllers/PointModeController.js

```diff
@@ -152,15 +152,15 @@ class PointModeController extends AnnotationModeController {
      * @return {void}
      */
     pointClickHandler(event) {
-        event.stopPropagation();
```

Comment:
I think this may have been moved here for something related to the mobile dialog. I would double check whether or not this causes any issues on tablets